src/Games/NexusMods.Games.StardewValley/Emitters/SMAPIModDatabaseCompatibilityDiagnosticEmitter.cs

```diff
@@ -116,7 +117,7 @@ public async IAsyncEnumerable<Diagnostic> Diagnose(Loadout.Model loadout, [Enume
                 yield return Diagnostics.CreateModCompatabilityAssumeBroken(
                     Mod: mod.ToReference(loadout),
                     ReasonPhrase: reasonPhrase ?? "it's no longer compatible",
-                    ModLink:  modPageUrls.GetValueOrDefault(manifest.UniqueID, DefaultWikiLink),
+                    ModLink: apiMods.GetValueOrDefault(manifest.UniqueID)?.NexusModsLink.ValueOr(() => Helpers.NexusModsLink) ?? Helpers.NexusModsLink,
```

Comment:
What was the `DefaultWikiLink`?

Reply:
`DefaultWikiLink` is now unused. But I think it made sense to use that if there was no Nexus link, since people would not be able to find the mod on Nexus anyways

Reply:
I would probably wrap this thing in a method, `GetLinkOrDefault` or something